packages/core/src/Types/ConfigTypes.ts

```diff
@@ -145,6 +145,11 @@ export namespace ConfigTypes {
      */
     dev?: boolean;
 
+    /**
```

Comment:
As before. This is unnecessary while users will inject WSS service manually to the container